src/core/src/pass/visualize_tree.cpp

```diff
@@ -181,6 +195,14 @@ bool ov::pass::VisualizeTree::run_on_model(const std::shared_ptr<ov::Model>& f)
 
     for (auto it = nodes.rbegin(); it != nodes.rend(); ++it) {
         auto& node = *it;
+        if (auto multi_subgraph_op = std::dynamic_pointer_cast<op::util::MultiSubGraphOp>(node)) {
+            for (size_t i = 0; i < multi_subgraph_op->get_internal_subgraphs_size(); ++i)
+                if (const auto& sub_graph = multi_subgraph_op->get_function(i))
+                    ov::pass::VisualizeTree(name_of_subgraph_file(multi_subgraph_op, m_name, i),
+                                            m_node_modifiers,
+                                            m_dot_only)
+                        .run_on_model(sub_graph);
+        }
```

Comment:
Is it possible to add the same functionality for `ov::pass::Serailize` (maybe as optional parameter)?

Reply:
Inner graphs of MultiSubgraph op are serialized in xml without problems. If you wish to serialize them separately -- you can just pass inner `ov::Model` to `ov::pass::Serailize`. I think Serialize pass should be kept as-is since it is exposed to customer in many ways.